components/BottomAppBar/src/private/MDCBottomAppBarLayer.m

```diff
@@ -0,0 +1,252 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCBottomAppBarLayer.h"
+
+#import "MDCBottomAppBarAttributes.h"
+#import "MaterialMath.h"
+
+@implementation MDCBottomAppBarLayer
+
++ (instancetype)layer {
+  MDCBottomAppBarLayer *layer = [super layer];
+  layer.fillColor = [UIColor whiteColor].CGColor;
+  layer.shadowColor = [UIColor blackColor].CGColor;
+
+  // TODO(junius): These shadow attributes will be updated once Google Material specs are finalized.
+  layer.shadowOpacity = 0.4f;
+  layer.shadowRadius = 4.f;
+  layer.shadowOffset = CGSizeMake(0, 2.f);
+  layer.needsDisplayOnBoundsChange = YES;
+  layer.contentsScale = 1.f;
```

Comment:
Shouldn't these scales be the `UIScreen.mainScreen.scale`?

Reply:
Done.